unit_scaling/scale.py

```diff
@@ -16,7 +16,7 @@
     import poptorch_experimental_addons as pea
 
     _poptorch_available = True
-except (ImportError, ModuleNotFoundError):  # pragma: no cover
+except ImportError:  # pragma: no cover
```

Comment:
Because I gather `ModuleNotFoundError` is a subtype of `ImportError` & tried a try-catch which seems to work accordingly; but I seem to remember you correcting me in the past about this one, so please shout if I'm wrong to change this!